mobile-app/src/i18n/locales/en.json

```
@@ -132,6 +147,53 @@
       "createBusiness": "Create Business"
     }
   },
+  "appliances": {
+    "applianceDetails": "Appliance Details",
+    "location": "Location",
+    "status": "Status",
```

Biome (1.9.4)

[error] The key status was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)